source/features/comment-on-draft-pr-indicator.tsx

```diff
@@ -17,19 +17,21 @@ function addDraftBanner(newCommentField: HTMLElement): void {
 	);
 }
 
+export const newCommentFieldSelector = [
+	'[input="fc-new_comment_field"]',
+	'[input^="fc-new_inline_comment_discussion"]',
+];
```

Comment:
Can you move this to selectors.ts? Then add an empty array for the tests, if allowed. Or `[0, 'some url']` to have `0 expected matches`